frontend/src/hooks/useGetBoardInfo.ts

```diff
@@ -103,10 +103,12 @@ export const useGetBoardInfoEffect = (): useGetBoardInfoInterface => {
             return res;
           })
           .catch((err) => {
-            const { code } = err;
-            errorCount++;
-            localBoardInfoFailedStatus = boardInfoPartialFailedStatusMapping(code);
-            setBoardInfoFailedStatus(localBoardInfoFailedStatus);
+            if (typeof err === 'object') {
```

Comment:
please use `err?.code`, since `typeof null === object // true`. Yes, this is JAVASCRIPT!

Reply:
fixed, thx